drools/src/main/java/com/datatorrent/drools/operator/StatelessDroolsOperator.java

```diff
@@ -43,22 +44,78 @@
 public class StatelessDroolsOperator extends BaseOperator implements ActivationListener<Context.OperatorContext>
```

Comment:
Have we finalized on the license for this operator. If so, please add appropriate license header.. for all files..